app/views/home/tos_faq.html.erb

```diff
@@ -285,14 +285,14 @@
+      At such time as we are able to host art and vids, we anticipate similar policies will apply to those mediums: Art and vids will be considered fannish even in cases where they do not directly depict characters and elements from or use footage from an existing media source if they were created in a fandom context for a fandom audience.  This may include such things as fanart that is intended to be for a particular canon but which does not contain readily identifiable canon characters or elements, art and vids produced as part of a fandom challenge, exchange, or charity event, illustrations of fanfiction, and vids that comment on a particular canon without using any clips from it (e.g., fan-produced videos set in a particular universe).
       </p>
       <p>
        We presume that, by posting the work to the Archive, the creator is making a statement that they believe it's a fanwork. As such, unless the work doesn't meet some other criterion, it will be allowed to remain.
       </p>
 <h5 class="heading" id="nonfiction">Can I archive nonfiction?</h5>
 <p>
-Fannish nonfiction, which includes what is called 'meta' by some fans, is allowed. Where we provide a specific function (search, bookmarking, challenges) we will ask you to use the specific methods we provide for those activities rather than create separate works. So, for example, a request for recommendations for particular kinds of fanworks would not be an appropriate work.  That search should be carried out by searching works and/or bookmarks.  A list of recommended works on a particular topic would also not be an appropriate work.  Recommendations should be done by using our bookmarking function.  A description of a challenge for other creators would also not be an appropriate work.  That should be carried out by using our challenge function.  
+Fannish nonfiction, which includes what is called 'meta' by some fans, is allowed. Where we provide a specific function (search, bookmarking, challenges) we will ask you to use the specific methods we provide for those activities rather than create separate works. So, for example, a request for recommendations for particular kinds of fanworks would not be an appropriate work.  That search should be carried out by searching works and/or bookmarks.  A list of recommended works on a particular topic would also not be an appropriate work.  Recommendations should be done by using our bookmarking function.  A description of a challenge for other creators would also not be an appropriate work.  That should be carried out by using our challenge function.
```

Comment:
Also two spaces at the end that have been stripped out. Don't know why Github hasn't highlighted them, since it's done it on another line in this diff.